src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -43,6 +43,11 @@ public class AddDependenciesToProjectJson : Task
 
         // External package dependency versions.
         public ITaskItem[] ExternalPackages { get; set; }
+
+        /// <summary>
+        /// Optional list of RIDs to exclude from the generated project.json.
+        /// </summary>
+        public ITaskItem[] ExcludedRuntimes { get; set; }
```

Comment:
These could be just `string[]` and then below you can just pass the, directly into the constructor for HashSet.